extensions/wikia/ExactTargetUpdates/tasks/ExactTargetRetrieveWikiHelper.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Wikia\ExactTarget;
+
+class ExactTargetRetrieveWikiHelper extends ExactTargetTask {
+
+	/**
+	 * Retrieve wiki data from ExactTarget by wiki id
+	 * @param int $iWikiId
+	 * @return array|null
+	 * @throws \Exception
+	 */
+	public function retrieveWikiDataById( $iWikiId ) {
+		$aProperties = [
+			'city_path',
+			'city_dbname',
+			'city_sitename',
+			'city_url',
+			'city_created',
+			'city_founding_user',
+			'city_adult',
+			'city_public',
+			'city_title',
+			'city_founding_email',
+			'city_lang',
+			'city_special',
+			'city_umbrella',
+			'city_ip',
+			'city_google_analytics',
+			'city_google_search',
+			'city_google_maps',
+			'city_indexed_rev',
+			'city_lastdump_timestamp',
+			'city_factory_timestamp',
+			'city_useshared',
+			'ad_cat',
+			'city_flags',
+			'city_cluster',
+			'city_last_timestamp',
+			'city_founding_ip',
+			'city_vertical'
+		];
+
+		$oHelper = $this->getWikiHelper();
+		$aApiParams = $oHelper->prepareWikiDataExtensionRetrieveParams( $aProperties, 'city_id', [ $iWikiId ] );
+
+		$oApiDataExtension = $this->getApiDataExtension();
+		$oWikiResult = $oApiDataExtension->retrieveRequest( $aApiParams );
+
+		if ( isset( $oWikiResult->OverallStatus ) && $oWikiResult->OverallStatus !== 'OK' ) {
+			throw new \Exception( $oWikiResult->OverallStatus );
+		}
+
+		if ( !empty( $oWikiResult->Results->Properties->Property ) ) {
+			$aProperties = $oWikiResult->Results->Properties->Property;
+			$aExactTargetWikiData = [];
+			foreach ( $aProperties as $value ) {
+				$aExactTargetWikiData[$value->Name] = $value->Value;
```

Comment:
Per coding conventions we're adding spaces around array contents and don't around arrays keys
https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax